src/plugins/data_source_management/public/components/utils.ts

```diff
@@ -143,3 +143,19 @@ export const getDefaultAuthMethod = (
 
   return initialSelectedAuthMethod;
 };
+
+export const extractRegisteredAuthTypeCredentials = (
+  currentCredentialState: { [key: string]: string },
+  authType: string,
+  authenticationMethodRegistery: AuthenticationMethodRegistery
+) => {
+  const registeredCredentials = {} as { [key: string]: string };
+  const registeredCredentialField =
+    authenticationMethodRegistery.getAuthenticationMethod(authType)?.crendentialFormField ?? {};
```

Comment:
This attribute name has a typo `cre*n*dentialFormField`.

@xinruiba

Reply:
Thanks for the comment.
Typo get resolved in this PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6122

Thanks again